superset-frontend/src/components/ErrorBoundary/index.tsx

```diff
@@ -56,6 +58,26 @@ export default class ErrorBoundary extends React.Component<
       const messageString = `${t('Unexpected error')}${
         firstLine ? `: ${firstLine}` : ''
       }`;
+      if (/Loading chunk [\d]+ failed/.test(firstLine)) {
```

Comment:
Are there some other ways to track which are needed and which are not?

Reply:
I think this problem should fixed using a different strategy where we invalidate cached frontend assets when necessary (ex: when the version changes).

Reply:
@mistercrunch any thought about it.

Reply:
idk (maybe) it seems the key is in the manifest and the manifest version (?) `superset/static/assets/manifest.json`. Maybe both the frontend/backend need to be aware of some sort of manifest SHA and have some sort of handshake / check around that specifically. Not sure how its generally handled. I believe at Preset we persist previous assets on a CDN for longer, but it may not be the best approach for everyone. 

I believe there's also something somewhere that checks to see whether you're still logged in, and redirects you to the login screen if you're not. Maybe this same ping could check for manifest/app version on top of that (?) Seems like a common thing in the webpack world, might be worth looking for best practices around this specific family of issues.

Reply:
Maybe someone has any advices about how to implement that?